functions.py

```diff
@@ -331,19 +336,20 @@ def get_to_base_unit_and_type_prefix(from_uri, query_uri):
             rdf:predicate rdf:type ;
             rdf:object geo:Feature .
     }
-    UNION
-    { ?l a prov:Location }
-    UNION
-    {
-        ?c2 rdfs:subClassOf+ prov:Location .
-        ?l a ?c2 .
-    }
-    UNION
-    {
-        ?s2 rdf:subject ?l ;
-            rdf:predicate rdf:type ;
-            rdf:object prov:Location .
-    } .
+#    UNION
```

Comment:
My best guess as to why this was here in the first place is because "overlaps" objects are treated as geo:Features but we want to exclude them from being returned as locations. I'm not sure though, @ashleysommer ?

Reply:
Are these computed ahead and cached? If so, I would prefer to see an explicit feature-type for overlaps. Maybe `loci:Overlap` ? Then _these_ can be explicitly excluded., rather than borrowing an unrelated feature-type.

Reply:
Yes I agree it shouldn't be a feature but I can't remember all the reasoning around it from our older discussions. I believe we can actually get rid of overlaps objects altogether because all that information should be elsewhere but @ashleysommer has more history and probably more insight on this. We need to break up scope here as well. It would be good to merge whatever is the minimum set of changes to support the new LDAPIs and deal with larger refactors, like overlaps, later.